node/pkg/watchers/evm/connectors/arbitrum.go

```diff
@@ -0,0 +1,42 @@
+// On Arbitrum we are unable to get blocks by transaction hash using the go-ethereum library
+// because it fails with "transaction type not supported". However, calling the underlying
+// eth_getBlockByHash directly works. The sole function of this connector is to implement
+// TimeOfBlockByHash using the raw connection.
+
+package connectors
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	ethCommon "github.com/ethereum/go-ethereum/common"
+)
+
+type ArbitrumConnector struct {
+	Connector
+}
+
+func NewArbitrumConnector(ctx context.Context, baseConnector Connector) (*ArbitrumConnector, error) {
+	connector := &ArbitrumConnector{Connector: baseConnector}
```

Comment:
nit: this doesn't need to return an error

Reply:
True, but all of the other new connector methods do, so I would like to keep the interface standard.

Besides, who knows what might be added later.